src/UKHO.ERPFacade.API/Controllers/WebhookController.cs

```diff
@@ -24,35 +24,37 @@ public class WebhookController : BaseController<WebhookController>
         private readonly IAzureTableReaderWriter _azureTableReaderWriter;
         private readonly IAzureBlobEventWriter _azureBlobEventWriter;
         private readonly ISapClient _sapClient;
-        private readonly ISapMessageBuilder _sapMessageBuilder;
+        private readonly IEncContentSapMessageBuilder _encContentSapMessageBuilder;
         private readonly IOptions<SapConfiguration> _sapConfig;
         private readonly ILicenceUpdatedSapMessageBuilder _licenceUpdatedSapMessageBuilder;
+        private readonly IRecordOfSaleSapMessageBuilder _recordOfSaleSapMessageBuilder;
 
         private const string CorrelationIdKey = "data.correlationId";
         private const string EncEventFileName = "EncPublishingEvent.json";
         private const string SapXmlPayloadFileName = "SapXmlPayload.xml";
         private const string LicenceUpdatedContainerName = "licenceupdatedblobs";
         private const string LicenceUpdatedFileName = "LicenceUpdatedEvent.json";
         private const string RecordOfSaleContainerName = "recordofsaleblobs";
-        private const string RecordOfSaleEventFileName = "RecordOfSaleEvent.json";
-        private const string RosXmlFilePath = "SapXmlTemplates\\RosSapRequest.xml";
+        private const string RecordOfSaleFileName = "RecordOfSaleEvent.json";
```

Comment:
```suggestion
        private const string RecordOfSaleEventFileName = "RecordOfSaleEvent.json";
```

Reply:
changed from RecordOfSaleFileName to RecordOfSaleEventFileName